Update vmware-walkthrough-prerequisites.md

sbkuehn opened the pull request:
VMware has not supported vSphere 5.1 since August 24, 2016. Other Azure Site Recovery docs have the right versions listed.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Update vmware-walkthrough-prerequisites.md

VMware has not supported vSphere 5.1 since August 24, 2016. Other Azure Site Recovery docs have the right versions listed.
